coronaqr.go

```diff
@@ -346,7 +346,7 @@ func (u *Unverified) Verify(certprov PublicKeyProvider) (*Decoded, error) {
 		}
 	}
 	if err := u.u.verify(expired, certprov); err != nil {
-		return nil, err
+		return u.u.decoded(), err
```

Comment:
Can you update the function comment to say that the decoded certificate is returned, but should be discarded, when verification fails?